src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -513,6 +515,35 @@ private static string[] GetDefaultLetterKeyDE(LetterKey letter)
             };
         }
 
+        // Greek
+        private static string[] GetDefaultLetterKeyEL(LetterKey letter)
+        {
+            return letter switch
+            {
+                LetterKey.VK_A => new string[] { "Α", "α" },
+                LetterKey.VK_B => new string[] { "Β", "β" },
+                LetterKey.VK_C => new string[] { "Χ", "χ" },
+                LetterKey.VK_D => new string[] { "Δ", "δ" },
+                LetterKey.VK_E => new string[] { "Ε", "ε", "Η", "η" },
+                LetterKey.VK_G => new string[] { "Γ", "γ" },
+                LetterKey.VK_I => new string[] { "Ι", "ι" },
+                LetterKey.VK_K => new string[] { "Κ", "κ" },
+                LetterKey.VK_L => new string[] { "Λ", "λ" },
+                LetterKey.VK_M => new string[] { "Μ", "μ" },
+                LetterKey.VK_N => new string[] { "Ν", "ν" },
+                LetterKey.VK_O => new string[] { "Ο", "ο", "Ω", "ω" },
+                LetterKey.VK_P => new string[] { "Π", "π", "Φ", "φ", "Ψ", "ψ" },
+                LetterKey.VK_R => new string[] { "Ρ", "ρ" },
+                LetterKey.VK_S => new string[] { "Σ", "σ" },
+                LetterKey.VK_T => new string[] { "Τ", "τ" },
+                LetterKey.VK_U => new string[] { "Υ", "υ" },
+                LetterKey.VK_X => new string[] { "Ξ", "ξ" },
+                LetterKey.VK_Y => new string[] { "Υ", "υ" },
+                LetterKey.VK_Z => new string[] { "Ζ", "ζ" },
```

Comment:
You don't need to include the uppercase letters, they will get capitalized automatically if you hold down <kbd>Shift</kbd> (or have CapsLock on) when activating Quick Accent

Reply:
You are right. But I assumed, that some people don't know if they want the capital letter or not. Phi and Psi for example. But for the consistency of the usage and code it would be smarter if there are only the small letters.